### DISCUSSION TITLE
Import IIS logs FTP and W3

### DISCUSSION THREAD

**Discussion by jazecap** [Security-Onion-Solutions/securityonion#3694] [Unsupported Versions]
I want to know if it is possible to import the IIS logs like the pcap file we can import, is there also an IIS log?.

**Comment by jazecap** [comment]
additional info, my IIS log I converted to CSV file and I successfully uploaded my file to elastic search, hopefully Security Onion also has, the question is how?.

![Import](https://user-images.githubusercontent.com/79244169/113056637-091f0e80-917a-11eb-8abc-92c461a5c5fd.PNG)

**Comment by weslambert** [comment]
We don't have an import function specifically for Security Onion, but it sounds like you can do as you mentioned, or use Filebeat, or the Logstash file input.

**Comment by jazecap** [comment]
Do you have an example config that I can base on?.

Best Regards
jhayvee

**Reply by Leche671** [reply]
Did you find a solution?
